src/backend/common/sentry.py

```diff
@@ -49,8 +49,9 @@ def init_sentry_client(dsn: Optional[str] = None) -> None:
             RedisIntegration(),
         ],
     )
-    with sentry_sdk.configure_scope() as scope:
-        scope.set_tag('aws_instance_ip', get_aws_instance_api())
+    if os.getenv('SKIP_AWS_CHECKING', 'False') != 'True':
```

Comment:
It's not bool, because it's env var. It's always string.

```
[stepan@soksanichenko mirrors.d]$ export VAR=True
[stepan@soksanichenko mirrors.d]$ python
Python 3.6.8 (default, Aug 24 2021, 23:39:28)
[GCC 8.5.0 20210514 (Red Hat 8.5.0-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.environ['VAR']
'True'
>>>
```

Reply:
This part is rewritten